### ISSUE TITLE
Deprecation in Symfony 6.4

### ISSUE THREAD

**Issue by craigh** [DamienHarper/auditor-bundle#381]
| Q                           | A
| ----------------------------| -----------------------
| `auditor-bundle` version    | dev-master
| PHP version                 |  8.2.13 
| Database                    | MySQL 8

#### Summary

Since symfony/routing 6.4: The "routing.loader.annotation" service is deprecated, use the "routing.loader.attribute" service instead. It is being referenced by the "DH\AuditorBundle\Routing\RoutingAnnotationLoader" service.

**Comment by b1rdex** [comment]
This issue makes `\DH\AuditorBundle\Routing\RoutingAnnotationLoader` unusable @ symfony 7

```
Cannot autowire service "DH\AuditorBundle\Routing\RoutingAnnotationLoader":
argument "$annotatedRouteController" of method "__construct()"
references class "Symfony\Bundle\FrameworkBundle\Routing\AnnotatedRouteControllerLoader"
but no such service exists.
```

**Comment by DamienHarper** [comment]
I close this issue since this should be fixed in `master` branch (PR #399).
Feel free to reopen if needed.
